dart-common/src/main/java/com/f2prateek/dart/common/AbstractDartProcessor.java

```diff
@@ -283,6 +289,72 @@ private void parseInjectExtra(Element element, Map<TypeElement, InjectionTarget>
     erasedTargetTypes.add(erasedTargetType);
   }
 
+  private boolean isParcel(TypeMirror type) {
+    List<Element> supportedTypes = getTypeElements(
+            new Class[]{Byte.class, Double.class, Float.class, Integer.class,
+                        Long.class, Character.class, Boolean.class, String.class});
+
+    List<Element> singleGenericCollections = getTypeElements(
+            new Class[]{List.class, ArrayList.class, LinkedList.class,
+                        Set.class, HashSet.class, SortedSet.class, TreeSet.class,
+                        LinkedHashSet.class});
```

Comment:
Nothing at the moment... you could add that to the `supportedTypes` as it's not a generic collection.  There are a few other misc types I didn't map like `Bundle` or `IBinder`.  Here's the list Parceler supports:  https://github.com/johncarl81/parceler/blob/master/parceler/src/main/java/org/parceler/internal/ParcelerModule.java#L219-L258